tools/lint/main_ni.c

```diff
@@ -215,6 +217,10 @@ help(int shortout)
     printf("  -J, --json-null\n"
             "                Allow usage of JSON empty values ('null') within input data\n\n");
 
+    printf("  -k, --ext-id\n"
```

Comment:
`ext-id` is okay but could be called better, `id` is not used much in YANG. Just `ext-inst` or `ext-instance` should be better, you are identifying a specific extension instance.